couchdb-setup.sh

```diff
@@ -47,6 +47,24 @@ else
   COUCHURL=http://$COUCHUSER:$COUCHPASSWORD@$HOST:$PORT
 fi
 
+# Adding attachment to resource
+insert_attachment() {
+    python -c "
+import urllib, json, sys, subprocess
+data = json.load(open('./design/resources/attachment.json'))
+for key in data:
+ id=key['doc_id']
+ image=key['filename']
+ command = '$COUCHURL/resources/'+id
+ response = urllib.urlopen(command)
+ jsondata = json.loads(response.read())
+ rev=jsondata['_rev']
+ putAttachment='curl -v -X PUT $COUCHURL/resources/'+id+'/'+image+'?rev='+rev+' --data-binary @'+image+' -H Content-Type:image/jpg'
```

Comment:
Let's also make `Content-Type` dynamic so we can add different kinds of files as well